cmd/neofs-adm/docs/subnetwork.md

```diff
@@ -0,0 +1,116 @@
+# Step-by-step NeoFS subnetwork creation
+
+This is a short guide on how to create and manage NeoFS subnetworks.
+This guide considers that side chain and inner ring (alphabet nodes)
+have already been deployed and side chain contains deployed `subnet`
+contract.
+
+## Prerequisites
+
+To follow this guide you need:
+- script hash of `subnet` contract in side chain (little-endian);
+- neo-go side chain RPC endpoint;
+- latest released version of [neofs-adm](https://github.com/nspcc-dev/neofs-node/releases);
+- private key of the NeoFS account;
+- public key of the Storage Node (optional);
+- public keys of the node and client administrators (optional);
+- owner IDs of the NeoFS users (optional).
+
+## Step 1: Subnet creation
+
+```shell
+$ neofs-adm morph subnet create \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/owner/private/key> \
+    --notary
+Create subnet request sent successfully. ID: 4223489767.
+```
+
+**NOTE:** use `--notary` only in notary-enabled environmental. You need to have sufficient notary deposit (not expired
+with enough GAS balance). This is the only one command that requires alphabet signature and, therefore, the only one
+command that needs `--notary` flag. Your subnet ID will differ from the example.
+
+Private key that has been passed with `--key` flag is the key of the owner of the just created subnetwork.
+
+You can check if your subnetwork was created successfully:
+
+```shell
+$ neofs-adm morph subnet get \
+    -r <side_chain_RPC_endpoint> \
+    -a <subnet_script_hash> \
+    --key </path/to/owner/private/key> \
```

Comment:
To get public info from the chain, you don't need a key, e.g. `morph dump-hashes` does not require wallet, so neither should it.

Reply:
https://github.com/nspcc-dev/neofs-node/pull/1021